tests/src/test/scala/com/nvidia/spark/rapids/ParquetScanSuite.scala

```diff
@@ -90,4 +90,10 @@ class ParquetScanSuite extends SparkQueryCompareTestSuite {
     frameFromParquetWithSchema("disorder-read-schema.parquet", StructType(Seq(
       StructField("c3_long", LongType),
       StructField("c1_int", IntegerType))))) { frame => frame }
+
+  testSparkResultsAreEqual("Test Parquet unsigned int: uint8, uint16, uint32",
+    frameFromParquet("unsigned-int.parquet"),
```

Comment:
Does this file cover the nested columns ? If not, better to add them.
e.g. array<unit8>, struct<a:unit16, b: unit8, c:unit32>

Reply:
yes, added a comment

Reply:
I mean we should have nested columns to be tested. Not only add the comments. We need to update the parquet file to involve some nested columns containing unsigned types.

Reply:
Did not catch up, test case updated, including nested columns, see code.